Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.12.3-alpine as base
+    RUN pip install poetry
```

Comment:
I've never seen this pattern of indenting a Dockerfile before, and I really like it!

Reply:
Oh interesting. I feel like I must have got it from somewhere but I could not tell you where